src/Module.Server/Common/Models/CrpgItemValueModel.cs

```diff
@@ -424,16 +424,25 @@ private float ItemToHeirloomLevel(ItemObject item)
     {
         if (item == null)
         {
+            InformationManager.DisplayMessage(new InformationMessage("one of the item was null, please report to namidaka"));
             return 0;
         }
 
-        if (!item.StringId.StartsWith("crpg_")

```

Comment:
Looks like you are reverting the last commit here

Reply:
solved